### ISSUE TITLE
Import Trakt Watchlist Shows

### ISSUE THREAD

**Issue by redmondking** [SchizoDuckie/DuckieTV#956]
**What build of DuckieTV are you using (Standlone / Chrome Extension (New Tab / Browser Action))**
Standalone

**What version of DuckieTV are you using (Stable 1.1.x / Nightly yyyymmddHHMM)**
Mac

**What is your Operating System (Windows, Mac, Linux, Android)**
v1.1.15

**Describe the problem you are having and steps to reproduce if available**
I am not able to get DuckieTV to sync from Trackt.tv. I have just recently created a Trackt account after getting DuckieTV and add shows to the watch lists but after authorizing DuckieTV it does not sync from my Trackt watchlists. Am I doing something wrong or is this and issue/bug?

**Comment by Js41637** [comment]
DuckieTV will only import episodes (and shows from those episodes) marked as seen or optionally only shows you've seen that are part of your trakt.tv collection. We do not use watchlists.

**Comment by redmondking** [comment]
Will there be an option in the feature to import from a watchlist? Means I now have to go through every show I want to add and select it as watched which is a hassle.

**Comment by Js41637** [comment]
Well what is a watchlist? Watchlist means you haven't watched it thus are not importing anything except the show itself in which case you can just add them from within DuckieTV

**Comment by redmondking** [comment]
Ok. Cool. Thanks. I see your point. You can close this issue.

**Comment by Js41637** [comment]
I understand the concept though. We should probably add shows that are on your watchlist. I think this might not have been a thing when this was originally added.

**Comment by Js41637** [comment]
Will see what @garfield69 thinks.

**Comment by redmondking** [comment]
Yeah that would be nice. I have used duckietv before and forgot to do a backup (laptop got damaged) so while waiting for a new laptop I created a trackt.tv and added all the shows I wanted to watch\already watching to the watchlist. I think this should be stated somewhere to not confuse future users. Thanks for the help.

**Comment by garfield69** [comment]
I have reviewed the Trakt.TV API docs, and it does appear that we could do a better job during the DuckieTV Trakt import.     
We should:
- fetch the Trakt *shows* on the watchlist , (equivalent to DuckieTV's shows that have not had any episodes downloaded yet),     
- then fetch the Trakt collected episodes, (equivalent to DuckieTV's episodes that have been marked as downloaded),     
- then finally fetch the Trakt watched episodes (equivalent to DuckieTV's episodes that have been marked as watched).
I've reworked this ticket as an accepted enhancement request.

**Comment by redmondking** [comment]
It would kinda of nice if we could also decide how duckietv syncs with traktv. I don't like the fact duckietv syncs tv shows i already watch into my collection. I use my collection for movies I actually own on DVD. It would better if a watched episode would use the add to history function of traktv.

**Comment by Js41637** [comment]
Watched episodes are already added to your history. I  think we only use collection if you add a show to DuckieTV or if you've downloaded it. Not sure though.

**Comment by redmondking** [comment]
Collection is used when a show is marked as downloaded on duckietv. Would be nice if I could turn that off. I use collection on traktv for movies only.

**Comment by garfield69** [comment]
Do you know that you can filter your collection on Trakt.tv to display movies only?

**Comment by redmondking** [comment]
Yes I know about the filter. This is just a secondary request. Not as important. Just to give users more control. 

P.S.
Thanks for this software. It has simplified my tv experience.

**Comment by effs** [comment]
While it’s possible to add shows in duckie itself, it’s at the time of writing much more convenient to add them from trakt.tv’s calendar especially when exploring, f.e. going through fall premieres/new shows. And adding them to watchlist is the cleanest way to do it in trakt.
